server_installation/topics/network/outgoing.adoc

```diff
@@ -184,19 +184,19 @@ file::
   This is what the trustore does.
   The keystore contains one or more trusted host certificates or certificate authorities.
   This truststore file should only contain public certificates of your secured hosts.
-  This is _REQUIRED_ if `disabled` is not true.
+  This is _REQUIRED_ if `enabled` is true.
 
 password::
   Password for the truststore.
-  This is _REQUIRED_ if `disabled` is not true.
+  This is _REQUIRED_ if `enabled` is true.
 
 hostname-verification-policy::
   `WILDCARD` by default.
   For HTTPS requests, this verifies the hostname of the server's certificate.
    `ANY` means that the hostname is not verified. `WILDCARD` Allows wildcards in subdomain names i.e.
   *.foo.com. `STRICT` CN must match hostname exactly.
 
-disabled::
-  If true (default value), truststore configuration will be ignored, and certificate checking will fall back to JSSE configuration as described.
-  If set to false, you must configure `file`, and `password` for the truststore.
+enabled::
+  If false (default value), truststore configuration will be ignored, and certificate checking will fall back to JSSE configuration as described.
```

Comment:
The default value is `true` if I'm not mistaken. Relevant code I look at for reference is 

`DefaultKeycloakSessionFactory`

    protected boolean isEnabled(ProviderFactory factory, Config.Scope scope) {
        if (!scope.getBoolean("enabled", true)) {
            return false;
        }
        if (factory instanceof EnvironmentDependentProviderFactory) {
            return ((EnvironmentDependentProviderFactory) factory).isSupported();
        }
        return true;
    }

and `Config.Scope`

        Boolean getBoolean(String key, Boolean defaultValue);